src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -526,6 +520,7 @@ ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
     }
     if (npudesc.has_value() && npudesc->compiler_dq) {
         config.emplace("NPUW_DQ_FULL", "NO");
+        config.emplace("NPU_COMPILATION_MODE_PARAMS", "enable-weights-dynamic-dequantization=true");
```

Comment:
@TolyaTalamanov Could this property be present in the config at this point?
Could it be overwritten later by someone?